Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      predefined:
+      - panw–auth-portal-exclude-list
+    - default: false
+      isArray: false
+      name: source_url
+      description: The source URL. If the type is IP, URL or Domain, value must be provided.
+    - auto: PREDEFINED
+      isArray: false
+      name: frequency
+      required: false
+      description: Frequency to check for updates.
+      predefined:
+      - five_minute
+      - hourly
+      - daily
+      - weekly
+      - monthly
+    - default: false
+      isArray: false
+      name: frequency_hour
+      description: The frequency hour. If the frequency argument is daily, weekly or monthly, value must be provided. Possible values are 00-23.
+    - auto: PREDEFINED
+      isArray: false
+      name: day_of_week
+      required: false
+      description: The day of the week. If the frequency argument is weekly or monthly, value must be provided.
+      predefined:
+      - monday
+      - tuesday
+      - wednesday
+      - thursday
+      - friday
+      - saturday
+      - sunday
+    - default: false
+      isArray: false
+      name: day_of_month
+      description: The day of the month. If the frequency argument is monthly, value must be provided. Possible values are between 1 and 31.
+    - default: false
+      isArray: false
+      name: exception_list
+      description: The user can exclude certain addresses from the list depending on the type of list.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Create a new dynamic list.
+    name: prisma-sase-external-dynamic-list-create
+    outputs:
+    - contextPath: PrismaSase.ExternalDynamicList.id
+      description: The external dynamic list id.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.name
+      description: The external dynamic list name.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.folder
+      description: The external dynamic list folder.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.description
+      description: The external dynamic list description.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.type
+      description: The external dynamic list type.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.source
+      description: The external dynamic list source.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.frequency
+      description: The external dynamic list frequency.
+      type: String
+  - arguments:
+    - default: false
+      isArray: false
+      name: id
+      required: true
+      description: The external dynamic list id.
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is this a folder location or a folder type/name?